### PR TITLE
Make Swift collector doc more consistent with other collectors

### PR DESCRIPTION
**Pull request by ddbeck**
https://github.com/buildkite/docs/pull/1661#pullrequestreview-1049497187 was a post-merge review; this PR expands on those changes a little.

### CODE REVIEW

**Comment by plaindocs** [comment]
Soz for the impatient merge @ddbeck 

You'd think GH could give you a "X users have pending reviews" kinda notification.

File: pages/test_analytics/swift_collectors.md.erb [hunk 1]
```diff
@@ -12,9 +12,17 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
...
 Before you start, make sure XCTest runs with access to [CI environment variables](/docs/test-analytics/ci-environments).
 
-1. [Create a test suite](https://buildkite.com/docs/test-analytics), and copy the _Buildkite Test Analytics API token_ you see
+1. [Create a test suite](https://buildkite.com/docs/test-analytics) and copy the test suite API token.
 
-1. Add the `test-collector-swift` repository to the `Package.swift` manifest, and add `BuildkiteTestCollector` to any test target requiring analytics:
+1. [Securely](/docs/pipelines/secrets) set the `BUILDKITE_ANALYTICS_TOKEN` secret on your CI to the API token from the previous step.
+
+   If you're testing an Xcode project, note that Xcode doesn't automatically pass environment variables to the test runner, so you need to add them manually.
+   In your test scheme or test plan, go to the _Environment Variables_ section and add the following key-value pair:
```
Comment (plaindocs) [inline pages/test_analytics/swift_collectors.md.erb:20]: Unless I'm mistaken, this isn't the Buildkite environment variables section, which is why I didn't capitalize it first time round. We could just say it looks better like this. :-p 

See also https://github.com/buildkite/docs/pull/1661#discussion_r928943201

**Review by plaindocs — APPROVED** [review]
Thanks for the fixes.

File: pages/test_analytics/swift_collectors.md.erb (hunk 1 above)
Comment (ddbeck) [reply pages/test_analytics/swift_collectors.md.erb:20]: I dug into this a little more. I think this is referring to an Xcode UI string, which is capitalized.

File: pages/test_analytics/swift_collectors.md.erb [hunk 2]
```diff
@@ -12,9 +12,17 @@ You can also upload test results by importing [JSON](/docs/test-analytics/import
...
+
+    * key: `BUILDKITE_ANALYTICS_TOKEN`
+    * value: the API token from the previous step
```
Comment (ddbeck) [inline pages/test_analytics/swift_collectors.md.erb:23]: Oh, I shouldn't have done this line change. This is wrong—fixing.

**Comment by ddbeck** [comment]
> You'd think GH could give you a "X users have pending reviews" kinda notification.

Yeah, it's a real oversight!

File: pages/test_analytics/swift_collectors.md.erb (hunk 2 above)
Comment (plaindocs) [reply pages/test_analytics/swift_collectors.md.erb]: Oh yeah, I missed it entirely. :-)

**Review by plaindocs — APPROVED** [review]

**Pull request merged**